duckdns/data/run.sh

```diff
@@ -83,15 +83,23 @@ while true; do
 
     if [[ ${ipv6} == *:* ]]; then
         if answer="$(curl -s "https://www.duckdns.org/update?domains=${DOMAINS}&token=${TOKEN}&ipv6=${ipv6}&verbose=true")" && [ "${answer}" != 'KO' ]; then
-            bashio::log.info "${answer}"
+            if [[ "${answer}" == *NOCHANGE* ]]; then
+   		bashio::log.debug "${answer}"
+	    else
+		bashio::log.info "${answer}"
+      	    fi
```

Comment:
Fix indention to use spaces only:
```suggestion
            if [[ "${answer}" == *NOCHANGE* ]]; then
                bashio::log.debug "${answer}"
            else
                bashio::log.info "${answer}"
            fi
```